src/wallet/wallet.cpp

```diff
@@ -2881,7 +2881,13 @@ std::shared_ptr<CWallet> CWallet::Create(WalletContext& context, const std::stri
         // Only descriptor wallets can be created
```

Comment:
In 2faf30612350fd90d9d3c44d1bb7b055addc8331 "wallet: don't import external keys at creation if blank"

Nit in commit message:
```diff
- For multisig setups than involve an external signer
+ For multisig setups that involve an external signer 
```